src/testingHelpers.tsx

```diff
@@ -202,8 +202,8 @@ export const defaultSnapshotItem: SnapshotItem = {
     'rpm.packagegroup': 20,
   },
   added_counts: {
-    'rpm.advisory': 3864,
-    'rpm.package': 17208,
+    'rpm.advisory': 100,
+    'rpm.package': 200,
```

Comment:
the tests failed because these numbers showed up twice in the table (one for delta, one for package count).  I just changed the numbers to match, but an alternative would be to find the element more precisely.